tools/project_manager/page_compare.php

```diff
@@ -0,0 +1,190 @@
+<?php
+$relPath="./../../pinc/";
+include_once($relPath.'base.inc');
+include_once($relPath.'misc.inc');
+include_once($relPath.'theme.inc');
+include_once($relPath.'Project.inc');
+include_once($relPath."DPage.inc"); // remove_formatting()
+
+require_login();
+
+$comparator = new Comparator();
+$comparator->get_data();
+$comparator->render();
+
+class Comparator
+{
+    function __construct()
+    {
+        global $PROJECT_STATES_IN_ORDER;
+
+        $this->L_round_options = array('P1', 'P2', 'P3', 'F1');
+        $this->R_round_options = array('F1', 'F2');
+        $this->state_index = array_flip($PROJECT_STATES_IN_ORDER);
+    }
+
+    function get_data()
+    {
+        $this->projectid = validate_projectID('project', @$_GET['project']);
+        $this->L_round_id = get_enumerated_param($_GET, "L_round_id", "P3", $this->L_round_options);
+        $this->R_round_id = get_enumerated_param($_GET, "R_round_id", "F1", $this->R_round_options);
+        $this->page_set = get_enumerated_param($_GET, "page_set", "all", array('left', 'right', 'all'));
+        $this->go_compare = isset($_GET['compare']);
+    }
+
+    function render()
+    {
+        global $pguser, $code_url;
+
+        $this->project = new Project($this->projectid);
+
+        $title = _('Compare pages with formatting removed');
+        $sub_title = $this->project->nameofwork;
+        output_header("$title: $sub_title", NO_STATSBAR);
+
+        echo "<h1>$title</h1>\n";
+        echo "<h2>$sub_title</h2>\n";
+
+        $state = $this->project->state;
+        $project_url = "$code_url/project.php?id=$this->projectid&amp;expected_state=$state";
+
+        $label = _("Return to Project Page");
+        echo "<p><a href='$project_url'>$label</a></p>\n";
+
+        if(!$this->project->check_pages_table_exists($warn_message))
+        {
+            echo "<p class='warning'>$warn_message</p>\n";
+            exit();
+        }
+
+        // draw the round selectors
+        echo "<form action='page_compare.php' method='GET'>
+            <input type='hidden' name='project' value='$this->projectid'>\n
+            <input type='hidden' name='compare'>\n";
+        // TRANSLATORS: %1$s and %2$s are selectors e.g. P3, %3$s etc. are radio button options
+        echo '<p>', sprintf(_('Compare %1$s to %2$s for %3$s %4$s %5$s'),
```

Comment:
I suspect this is going to be challenging to translate well. Moreover "My first" and "My second" are really confusing labels since they really mean "use all of the pages that I proofread in the [first|second] round" (and I had to read the code to figure that out). This would probably be better as a small form with labels rather than a sentence. Perhaps something like:
```
Compare rounds:
Round 1   [P3]
Round 2   [F1]
Show      o All pages
          o Pages I worked on in Round 1
          o Pages I worked on in Round 2
```

Reply:
It would probably be possible to add some javascript to change the text but seems like adding more complexity.
So it would be like
Compare rounds:   [P3]   [F1]
Show      o All pages
          o Pages I worked on in P3
          o Pages I worked on in F1

Reply:
I don't think you need to do any fancy javascript, I just think you need a form/table with labels you can use to refer to things. Much like we do with the review_work.php page (ie: "work round" and "compare round").